src/Calculator/App.xaml

```diff
@@ -1968,16 +2008,20 @@
                                         </VisualState>
                                         <VisualState x:Name="ButtonCollapsed"/>
                                     </VisualStateGroup>
-
                                 </VisualStateManager.VisualStateGroups>
 
+                                <Rectangle Grid.Column="0"
+                                           Width="10"
+                                           Fill="{TemplateBinding EquationColor}"
+                                           Visibility="{ThemeResource RectangleVisibility}"/>
                                 <ToggleButton x:Name="EquationButton"
+                                              Grid.Column="1"
                                               MinWidth="44"
                                               MinHeight="44"
                                               VerticalAlignment="Stretch"
-                                              Background="{TemplateBinding EquationColor}"
-                                              Foreground="{StaticResource SystemChromeWhiteColor}"
-                                              BorderBrush="{TemplateBinding EquationColor}">
+                                              Background="{ThemeResource EquationButtonBackground}"
+                                              Foreground="{ThemeResource EquationButtonForeground}"
+                                              BorderBrush="{ThemeResource EquationBoxBorderBrush}">
```

Comment:
BorderBrush is not consistent with the default style when the button in the following states: Hover (should be green with High Contrast #1 and White when Checked (CheckedHover and CheckedPressed are ok)